test-integration/jdl-samples/ms-mf-vue-consul-oauth2-mysql-memcached/blog-store.jdl

```diff
@@ -27,7 +27,7 @@ application {
     creationTimestamp 1617901618886
     jwtSecretKey "ZjY4MTM4YjI5YzMwZjhjYjI2OTNkNTRjMWQ5Y2Q0Y2YwOWNmZTE2NzRmYzU3NTMwM2NjOTE3MTllOTM3MWRkMzcyYTljMjVmNmQ0Y2MxOTUzODc0MDhhMTlkMDIxMzI2YzQzZDM2ZDE3MmQ3NjVkODk3OTVmYzljYTQyZDNmMTQ="
     packageName com.okta.developer.gateway
-    prodDatabaseType mysql
+    prodDatabaseType postgresql
```

Comment:
Why does this need to change?

Reply:
Mysql uses r2dbc-mariadb driver:
![7C4FB65E-5AE3-4CA8-8BD7-D163156968C5](https://user-images.githubusercontent.com/1417183/202485043-8b06eec6-c687-42f4-a59c-cae32a1e6332.jpeg)

Reply:
Thanks for the clarification.